core/src/test/java/com/google/zxing/oned/Code128WriterTestCase.java

```diff
@@ -32,14 +35,18 @@
   private static final String FNC3 = "10111100010";
   private static final String FNC4 = "10111101110";
   private static final String START_CODE_B = "11010010000";
+  private static final String START_CODE_C = "11010011100";
+  private static final String SWITCH_CODE_B = "10111101110";
   public static final String QUIET_SPACE = "00000";
   public static final String STOP = "1100011101011";
 
   private Writer writer;
+  private Code128Reader reader;
 
   @Before
   public void setup() {
     writer = new Code128Writer();
+    reader = new Code128Reader();
```

Comment:
This is mostly a test of encoding, but yeah seems OK if this needs some verification